app/scripts/models/auth_brokers/fx-firstrun-v1.js

```diff
@@ -26,6 +28,25 @@ define(function (require, exports, module) {
       VERIFICATION_COMPLETE: 'verification_complete'
     },
 
+    _isInAnIframe () {
+      return new Environment(window).isFramed();
```

Comment:
check if this needs to be `this._window`

Also tests need to be adjusted from app-start into this file

Reply:
ok :+1: